src/content/en/config/02-advanced.md

```diff
@@ -61,6 +58,17 @@ net.ipv4.tcp_congestion_control = hybla
 
 Of course, remember to execute `sysctl -p` to reload the config at runtime.
 
+```
+wget --no-check-certificate https://github.com/teddysun/across/raw/master/bbr.sh && chmod +x bbr.sh && ./bbr.sh
```

Comment:
Why do you think it's acceptable to run an arbitrary foreign script with root privilege?